week08_inference_software/03-triton/README.md

```diff
@@ -0,0 +1,4 @@
+More
```

Comment:
Maybe it's best to have a short guide at the beginning of each README outlining the necessary steps? E.g., "first launch script A, then launch script B, then build the image and run command C". Otherwise, the order of operations it a bit unclear